src/indexers/vault-compound-indexer.ts

```diff
@@ -0,0 +1,61 @@
+import { getDataMapper } from '../aws/dynamodb.utils';
+import { Chain } from '../chains/config/chain.config';
+import { constructVaultModel } from './indexer.utils';
+import { SUPPORTED_CHAINS } from '../chains/chain';
+import { BadgerSDK, RegistryVault } from '@badger-dao/sdk';
+import { VaultCompoundModel } from '../aws/models/vault-compound.model';
+
+export async function captureVaultData() {
+  for (const chain of SUPPORTED_CHAINS) {
+    const sdk = await chain.getSdk();
+    let registryVaults: RegistryVault[] = [];
+
+    try {
+      registryVaults = await sdk.vaults.loadVaults(true);
+    } catch (_) {
+      console.error(`Registry is not defined for ${chain.name}`);
+    }
+
+    if (registryVaults.length === 0) continue;
+
+    // update vaults from chain
+    await Promise.all(registryVaults.map(async (vault) => compoundVaultData(chain, sdk, vault)));
+
+    // update isProduction status, for vaults that were already saved in ddb
+    const prdVaultsAddrs = registryVaults.map((v) => v.address.toLowerCase());
+
+    const mapper = getDataMapper();
+    const query = mapper.query(
+      VaultCompoundModel,
+      { chain: chain.network },
+      { indexName: 'IndexVaultCompoundDataChain' },
+    );
+
+    try {
+      for await (const compoundVault of query) {
+        if (!prdVaultsAddrs.includes(compoundVault.address.toLowerCase())) {
```

Comment:
let's keep all vaults supported from `devGovernance` and `strategistGuild`

Reply:
thsi can be read from the registry

Reply:
basically, we are utilizing `sdk.vaults.loadVaults` which uses `getProductionVaults` and this is `productionVaults` in registry contract https://github.com/jack-the-pug/badger-registry/blob/2.0/contracts/BadgerRegistry.sol#L51, so we will have all prod vaults, we just need this marker to understand, if vault still presence in this map